src/scripts/healthsystem/equipment/equipment_catalogue.py

```diff
@@ -0,0 +1,102 @@
+import argparse
+from pathlib import Path
+
+import pandas as pd
+
+from tlo.analysis.utils import extract_results
+
+
+def get_annual_equipment_declarations_by_levels(results_folder: Path) -> pd.DataFrame:
+    """Return pd.DataFrame gives the simulated annual equipment declaration by facility levels for each simulated
+    year."""
```

Comment:
Perhaps also say that we require the `healthsystem.summary` logger to have been set at the level of `logger.INFO` or higher.